docs/getting_started/02_breakdown.md

```diff
@@ -23,11 +23,40 @@ commands, you would get a minimal Nargo project of the following structure:
 The source directory _src_ holds the source code for your Noir program. By default only a _main.nr_
 file will be generated within it.
 
-_Prover.toml_ is used for specifying the input values for executing and proving the program. You can specify `toml` files with different names by using the `--prover-name` or `-p` flags, see the [Prover](#provertoml) section below. Optionally you may specify expected output values for prove-time checking as well. 
+### Prover.toml
+
+_Prover.toml_ is used for specifying the input values for executing and proving the program. You can specify `toml` files with different names by using the `--prover-name` or `-p` flags, see the [Prover](#provertoml) section below. Optionally you may specify expected output values for prove-time checking as well.
+
+### Verifier.toml
 
 _Verifier.toml_ contains public in/output values computed when executing the Noir program.
 
-_Nargo.toml_ contains the environmental options of your project.
+### Nargo.toml
+
+_Nargo.toml_ contains the environmental options of your project. It contains a "package" section and a "dependencies" section.
+
+#### Package section
+
+The package section requires a number of fields including:
+
+- name - the name of the package
+- type - can be "bin" or "lib" to specify whether its a binary or library
+- authors
+- compiler_version - specifies the version of the compiler to use
```

Comment:
This actually does nothing currently.

Reply:
ill make a note that it isnt used currently, but plans to be